tracking/src/main/java/org/hps/recon/tracking/kalman/KalmanPatRecDriver.java

```diff
@@ -541,7 +558,22 @@ private ArrayList<KalTrack>[] prepareTrackCollections(EventHeader event, List<Tr
                 trackIntersectsRelations.add(new BaseLCRelation(intersectData, KalmanTrackHPS));
 
 
-                /*
+		//Add the Kinks
+		layers = new ArrayList<Integer>();
+		List<Double> Xkinks = new ArrayList<Double>();
```

Comment:
This loop should be over the clusters as well...please use the millipede id in the "layers" list.